docs/source/challenge-doc.rst

```diff
@@ -110,10 +119,29 @@ Observation Space
 
     - Hand passing error measures the distance between the MPL and the object
 
-    - The manipulated object has full 6 degrees of freedom, its state described as a 7 dimensional value in position + quaternion format  it is defined by a "`freejoint <https://mujoco.readthedocs.io/en/stable/XMLreference.html#body-freejoint>`__"
+    - The manipulated object has full 6 degrees of freedom, its state described as a 7 dimensional value in position + quaternion format. Details can be found in "`freejoint <https://mujoco.readthedocs.io/en/stable/XMLreference.html#body-freejoint>`__"
+
+
+
+
+**Variation on Object Properties**
+The geometry, mass, and friction of the object will reset at the start of each episode. 
```

Comment:
@Balint-H: this can be constraining. Let's rephrase it a bit to maintain our flexibility 

**Variation on Object Properties**
Both the geometry and physical properties of the object as well as the environment can reset at the start of each episode. Provided below is an example of how real-world scenarios might be captured during evaluations.

Reply:
@raku-slyu Could you implement these changes when you finalize the docs?

**Variation on Object Properties**
Both the geometry and physical properties of the object as well as the environment can be sampled at the start of each episode to provide variability in the task. Provided below is an example of how real-world scenarios is captured in the test environments we provide:

    - Object scale: a +- change between 0% - 5%, 0% - 10% scale variations in respective geom directions 
    - Object Mass: an upper/lower bound of +-50 gms
    - Object Friction: a +- change between 0 - 0.1, 0 - 0.001, 0 - 0.00002 from nominal value: [1.0, 0.005, 0.0001] in respective geom direction
    
Note that these distributions may be different in the final evaluation environment. Try to maintain the performance of your policies in as wide a range as possible.